src/components/Login.js

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable react/no-unescaped-entities */
+import React, { useState, useEffect } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { signinDetails } from '../store/userSlice';
+import './Login.css';
+
+function Login() {
```

Comment:
- [ ] When I login with any user, this happens:

![image](https://user-images.githubusercontent.com/57421823/207289936-123b8a06-6858-4488-8e09-41077872f26e.png)

Kindly correct this bug to make sure your app works as intended 👍